mbed_lstools/lstools_base.py

```diff
@@ -69,6 +69,12 @@ class MbedLsToolsBase(object):
     DETAILS_TXT_NAME = 'DETAILS.TXT'
     MBED_HTM_NAME = 'mbed.htm'
 
+    VENDOR_ID_DEVICE_TYPE_MAP = {
+        '0483': 'stlink',
```

Comment:
Can we rely on to these thtee VID in every boards now? what happens if VID does not match to some?

Reply:
Yes I believe we can, especially on anything released within the past few years. If the VID does not match it will default to the 'daplink' style detection (just like it used to).